### ISSUE TITLE
Is there a way to change the transparency level of the semi-transparent mode?

### ISSUE THREAD

**Issue by citbr** [germanfr/cinnamon-transparent-panels#43]
I would like just a small hint of opacity, but the semi-transparent mode is far too much.  Is there a way to specify the opacity level? Ideally I think I'd like somewhere around 5% to 20% opacity.

In future versions it would be great if there were more choices, maybe a percentage range of 100% transparent to 5% transparent in 5% intervals, selected by a drop-down menu.  Thanks for your consideration.

**Comment by germanfr** [comment]
Duplicate of #30

**Comment by citbr** [comment]
I apologize for posting a duplicate issue.  At the same time you appear to be confused about why people might want other transparency levels, so please allow me to explain a specific use case.  If you are trying to emulate the look of MacOS, for example by trying to follow the instructions at https://www.youtube.com/watch?v=DMs7DX3Um9E you find that using the latest version of Linux Mint it doesn't work right if you want a light theme (rather than dark as the author of that video seems to prefer) AND you are using the latest version of Linux Mint.  Specifically he has you install the WhiteSur theme from https://github.com/vinceliuice/WhiteSur-gtk-theme and originally I was using the "WhiteSur Light" theme which is a light theme but it also turns the text in the top panel white, making it very hard to read (you move the panel to the top so you can put the Plank dock at the bottom).  The author of the theme claims that one of the four variations of the theme, WhiteSur-alt, will give you a semi-transparent panel with black text but that is not how it works in Linux Mint.  Only his WhiteSur-solid variation gives black text but at the name implies, that gives the panel a non-transparent background.  So Since I need the black text to make it readable, I turned to your extension hoping I could restore a mostly transparent look.

If you look at the top menu bar in this image, this is more or less the effect I am trying to achieve: https://sm.pcmag.com/pcmag_in/photo/8/86-32/86-32_12xa.png

Your semi-transparent mode gives a much darker result than this and makes the black text fade, sort of the opposite problem from the white text of the original theme.

Basically it appears that when you go semi-transparent you are making the panel darker which is not what I am trying to achieve.  I hope that explains the issue.
